tests/core/math/test_transform_2d.h

```diff
@@ -45,6 +45,62 @@ Transform2D identity() {
 	return Transform2D();
 }
 
+Transform2D looking_at_old(Transform2D T, const Vector2 &p_target) {
+	Transform2D return_trans = Transform2D(T.get_rotation(), T.get_origin());
+	Vector2 target_position = T.affine_inverse().xform(p_target);
+	return_trans.set_rotation(return_trans.get_rotation() + (target_position * T.get_scale()).angle());
+	return return_trans;
+}
+
+TEST_CASE("[Transform2D] Looking at") {
+	Transform2D T = Transform2D(Vector2(1, 2), Vector2(3, 4), Vector2(5, 6));
+
+	Vector2 target = Vector2(0, 0);
+	Transform2D T_looking_target = T.looking_at(target);
+
+	// Check that the inverse maps target to the x-axis.
+	CHECK(Math::is_equal_approx(T_looking_target.affine_inverse().xform(target).y, 0));
```

Comment:
Checks involving standalone approximate float comparisons should use ` == doctest::Approx(` so that the test suite prints out helpful values.